scripts/dqlite/scripts/musl/install-if-missing.sh

```diff
@@ -8,9 +8,22 @@ PATH=${PATH}:${MUSL_BIN_PATH} which musl-gcc >/dev/null || { echo "Installing re
 
```

Comment:
I think my problem is at line 7, it assumes that `install` does the same setup as lines 11-22 below and it doesn't.

removing `exit 0;` from line 7 makes my run happy. 

Though I don't understand why the check at 11 fails if musl is installed but _deps/musl... hasn't been created.

Reply:
Whoops, I forgot to symlink on first install, so it works the next time around.